grails-app/views/layouts/generic.gsp

```diff
@@ -22,9 +22,9 @@
         // initialise plugins
         jQuery(function(){
             // autocomplete on navbar search input
-            jQuery("form#search-form-2011 input#search-2011, form#search-inpage input#search, input#search-2013").autocomplete('http://bie.ala.org.au/search/auto.jsonp', {
+            jQuery("form#search-form-2011 input#search-2011, form#search-inpage input#search, input#search-2013").autocomplete('http://bie.ala.org.au/search/auto.json', {
```

Comment:
Parameterising the bie URL here would be useful so we can switch to the test system easily (This is a broad issue for this URL across ALA websites but we can still fix it one instance at a time)

Reply:
Yes, should be - will fix

Reply:
Although this view is not used anymore (even by LA hubs) - maybe it should be deleted?

Reply:
Deleting is always useful when things drop out of use. When I was working on the hubs it was confusing having multiple views and only one of them was able to be used and if you got it wrong it was a bit confusing but it didn't necessarily break everything.

Reply:
deleted